src/lib/y2network/interface_config_builders/wireless.rb

```diff
@@ -0,0 +1,98 @@
+require "yast"
+require "forwardable"
+require "y2network/config"
+require "y2network/interface_config_builder"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    # Builder for wireless configuration. Many methods delegated to ConnectionConfig::Wireless
+    class Wireless < InterfaceConfigBuilder
+      extend Forwardable
+      include Yast::Logger
+
+      def initialize(config: nil)
+        super(type: InterfaceType::WIRELESS, config: config)
+      end
+
+      def mode
+        select_backend(
+          @config["WIRELESS_MODE"],
+          @connection_config.mode
+        )
+      end
+
+      def mode=(mode)
```

Comment:
What about documenting these methods? At least the return type and the expected params.